features/style-attributes.yml

```diff
@@ -0,0 +1,5 @@
+name: Style attributes
+description: The `style` global HTML attribute applies inline CSS styling declarations to individual elements.
+spec: https://html.spec.whatwg.org/multipage/dom.html#the-style-attribute
```

Comment:
I guess this should be in a CSS or HTML group, or both?

Reply:
Good catch. If this isn't a candidate to be in both, I'm not sure what would be 😄